ansible_collections/arista/cvp/plugins/modules/cv_facts.py

```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/python
+# pylint: skip-file
```

Comment:
so we skip this because there is much to fix? maybe lets make a PR to address it later?

Reply:
not fixing because this is V1 and we are going to deprecate it soon.

Reply:
ok makes sense :)